source/graphql/migrate-hasura.txt

```diff
@@ -129,73 +90,32 @@ GraphQL API for you.
       .. image:: /images/migration/connection-string.png
          :alt: Get Atlas clust connection string.
 
-   .. step::
-
-      Go back to the Hasura Cloud dashboard. On the Connect Existing Database
-      for MongoDB page, enter the name of the MongoDB database you want to
-      connect to and the connection string you copied from the previous step.
-
-      .. image:: /images/migration/add-connection-string.png
-         :alt: Add Atlas connection string to Hasura project.
-
-   .. step::
-
-      Click :guilabel:`Connect Database`.
-      You are finished setting up the connection between Hasura and MongoDB,
-      each hosted on their respective Cloud instances.
-
-.. _hasura-migrate-schema:
-
-Migrate GraphQL Schema
-----------------------
-
-Migrate your MongoDB Atlas GraphQL schema to Hasura's GraphQL schema. This
-involves defining your types, queries, mutations, and subscriptions in the
-Hasura Console or the Hasura CLI.
-
-.. procedure::
-
-   .. step::
-
-      In the Hasura console click the Data tab and select your MongoDB database
-      from the left hand pane.
-
-      You'll see all the collections from your MongoDB database.
-
-      .. image:: /images/migration/collections.png
-         :alt: Atlas collections are listed in a table.
-
-   .. step::
-
-      To generate a schema for a collection, click on the :guilabel:`Track`
-      button next to the desired collection.
-
-      You have three options to infer the schema of a collection:
+.. _hasura-create:
 
-      - Infer the schema from a sample document you copy and paste from your
-        collection.
-      - :guilabel:`Use Database Schema` allows you to use your database's
-        validation schema if one is present in your database (a GraphQL schema
-        will be automatically generated for you).
-      - :guilabel:`Use Existing Logical Model` allows you to select a previously
-        created logical model.
+Create a new project and connect to Hasura
+------------------------------------------
 
-      .. image:: /images/migration/generate-schema.png
-         :alt: Generate a new schema by using a sample document.
+Please visit the `Hasura docs <https://hasura.io/docs/3.0/getting-started/quickstart>`__
+to do the following steps:
```

Comment:
Nit: we don't generally use softening language like this in the docs. Instead of "Please visit" - maybe we just say:
```suggestion
The `Hasura docs <https://hasura.io/docs/3.0/getting-started/quickstart>`__ contain
the details to perform the following steps:
```